pages/signup.js

```diff
@@ -418,6 +418,10 @@ export default function Signup(props) {
             id="reset"
             custom="block font-body hover:text-canada-footer-hover-font-blue text-canada-footer-font underline mb-5"
             type="reset"
+            onClick={useEffect(() => {
```

Comment:
🤔 Is there a specific reason we're using a react hook in the onClick? Have never seen it used this way before.

Reply:
@will0684 I tried doing a function to call on onClick, but I had a error saying document is not defined, that's why I put it directly in the onClick.